grails-app/controllers/org/pih/warehouse/user/UserController.groovy

```diff
@@ -20,6 +20,7 @@ import org.pih.warehouse.core.User
 import javax.imageio.ImageIO as IIO
 import javax.swing.*
 import grails.gorm.transactions.Transactional
+import java.util.List
```

Comment:
This error
```GroovyCastException: Cannot cast object '[Requestor]' with class 'java.util.ArrayList' to class 'java.awt.List' due to: groovy.lang.GroovyRuntimeException: Could not find matching constructor for: java.awt.List(org.pih.warehouse.core.Role)```
was caused by the fact that we used `List` from `java.awt` instead of `java.util`

Reply:
@jmiranda I have not noticed this has not been removed. The reason I merged it alone, was that in the morning Katarzyna had problem with obdev3 not working and we thought a merge could help, before Artur is online (he starts work later than us). I wouldn't have merged it if it hasn't been the case. Sorry about that.